SNEWS_PT/snews_pub.py

```diff
@@ -61,6 +61,7 @@ def send(self, messages):
 
         """
         for message in messages:
+            print(message)
```

Comment:
we don't need a `print` statement since we already have `display_message()`

Reply:
you are right, this is a relic from testing 😄 I'll get rid of it

Reply:
done and committed 💯